megatron/model/utils.py

```diff
@@ -41,7 +41,7 @@ def init_(tensor):
 
 def attention_mask_func(attention_scores, attention_mask):
     args = get_args()
-    if args.curriculum_learning:
+    if args.curriculum_learning or args.data_efficiency_curriculum_learning:
```

Comment:
Yes args.curriculum_learning is the legacy CL. Unfortunately we cannot merge them because the usage and implementation are different. To better distinguish them I changed args.curriculum_learning to args.curriculum_learning_legacy.

Reply:
Sounds good.